dataset_configs/armenian/audio_books.yaml

```diff
@@ -0,0 +1,59 @@
+processors_to_run: "0:"
+workspace_dir: ???
+
+processors:
+  - _target_: sdp.processors.CreateInitialManifestByExt
```

Comment:
So this assumes there is some kind of pre-processing step to download the original files? Can it be included in the first step, so that it's fully automated? If not, you should at least describe this in the docs for this config.

Reply:
The pre-processing step is a manual one.